source/index.html

```diff
@@ -355,48 +356,61 @@ <h2 class="visually-hidden">Контакты</h2>
+                width="320"
+                height="200"
+                style="border: 0;"
+                allowfullscreen=""
+                loading="lazy"></iframe>
+
+        <!-- <img class="contacts__map-img"
              src="img/map.png"
              width="320"
              height="200"
-             alt="Карта." />
+             alt="Карта." /> -->
       </div>
     </section>
   </main>
 
-  <footer class="page__page-footer page-footer container">
-    <a class="page-footer__logo"
-       href="index.html">
-      <span class="visually-hidden">Pink</span>
-    </a>
-
-    <ul class="page-footer__social-list">
-      <li class="page-footer__social-item">
-        <a class="page-footer__social-link page-footer__social-link--twitter"
-           href="#">
-          <span class="visually-hidden">Twitter</span>
-        </a>
-      </li>
-      <li class="page-footer__social-item">
-        <a class="page-footer__social-link page-footer__social-link--facebook"
-           href="#">
-          <span class="visually-hidden">Facebook</span>
-        </a>
-      </li>
-      <li class="page-footer__social-item">
-        <a class="page-footer__social-link page-footer__social-link--youtube"
-           href="#">
-          <span class="visually-hidden">YouTube</span>
-        </a>
-      </li>
-    </ul>
-
-    <a class="page-footer__copyright"
-       href="https://htmlacademy.ru/intensive/adaptive">
-      <span class="page-footer__copyright-text"> Разработано</span>
-      <span class="visually-hidden">HTML Academy</span>
-    </a>
-  </footer>
+  <div class="container">
+    <footer class="page__page-footer page-footer ">
+
+      <a class="page-footer__logo"
+         href="index.html">
+        <span class="visually-hidden">Pink</span>
+      </a>
+
+      <ul class="page-footer__social-list">
+        <li class="page-footer__social-item">
+          <a class="page-footer__social-link page-footer__social-link--twitter"
+             href="#">
+            <span class="visually-hidden">Twitter</span>
+          </a>
+        </li>
+        <li class="page-footer__social-item">
+          <a class="page-footer__social-link page-footer__social-link--facebook"
+             href="#">
+            <span class="visually-hidden">Facebook</span>
+          </a>
+        </li>
+        <li class="page-footer__social-item">
+          <a class="page-footer__social-link page-footer__social-link--youtube"
+             href="#">
+            <span class="visually-hidden">YouTube</span>
+          </a>
+        </li>
+      </ul>
+
+      <a class="page-footer__copyright"
+         href="https://htmlacademy.ru/intensive/adaptive">
+        <span class="page-footer__copyright-text"> Разработано</span>
+        <span class="visually-hidden">HTML Academy</span>
+      </a>
+    </footer>
+  </div>
+
+  <script src="../js/menu.js"></script>
```

Comment:
```suggestion
  <script src="js/menu.js"></script>
```

Неверно указан относительный путь. HTML файлы находятся в корне сайта, а значит символы `..` (переход на один уровень вверх) не нужны.